src/Kiota.Builder/Writers/Go/CodeMethodWriter.cs

```diff
@@ -447,14 +447,16 @@ private void WriteRequestBuilderBody(CodeClass parentClass, CodeMethod codeEleme
             writer.WriteLines($"{RequestInfoVarName}.UrlTemplate = {GetPropertyCall(urlTemplateProperty, "\"\"")}",
                         $"{RequestInfoVarName}.PathParameters = {GetPropertyCall(urlTemplateParamsProperty, "\"\"")}",
                         $"{RequestInfoVarName}.Method = {conventions.AbstractionsHash}.{codeElement.HttpMethod?.ToString().ToUpperInvariant()}");
+            if(codeElement.AcceptedResponseTypes.Any())
+                writer.WriteLine($"{RequestInfoVarName}.Headers[\"Accept\"] = \"{string.Join(", ", codeElement.AcceptedResponseTypes)}\"");
```

Comment:
So if the `AcceptedResponseTypes` has `application/json` and `text/xml` then the response content type could be any of the 2, right? Maybe we should use response content type in the paging service instead of the request Accept header value in this case.